### PR TITLE
Convert all layer-related hyphenated vars to camelCase

### PR DESCRIPTION
**Pull request by juliexxia**
specifically:

asset-type -> assetType
ee-name -> eeName
display-name -> displayName
display-on-load -> displayOnLoad
color-function -> colorFunction
current-style -> currentStyle

and

layers -> layerArray

### CODE REVIEW

File: cypress/integration/unit_tests/checkbox_test.js [hunk 1]
```diff
@@ -15,52 +15,52 @@ const colorProperties = {
 };
 const mockFirebaseLayers = [
   {
-    'ee-name': 'asset0',
-    'asset-type': LayerType.FEATURE_COLLECTION,
-    'display-name': 'asset0',
-    'display-on-load': true,
-    'color-function': colorProperties,
+    'eeName': 'asset0',
```
Comment (janakdr) [inline cypress/integration/unit_tests/checkbox_test.js]: nit: I forgot to do this in my PR, but we can actually get rid of the quotes around the properties now in these object initializations.

File: cypress/integration/unit_tests/color_function_util_test.js [hunk 2]
```diff
@@ -99,54 +99,59 @@ describe('Unit tests for color function utility', () => {
...
 
     // switch to continuous
     $('#CONTINUOUS-radio').trigger('change');
     expectOneFirebaseWrite();
     const continuousPropertyPicker = $('#continuous-property-picker');
-    expect(getColorFunction()['current-style']).to.equal(0);
-    expect(getColorFunction()['color']).to.equal('red');
+    let currentStyle;
+    let color;
+    ({currentStyle, color} = getColorFunction());
```
Comment (janakdr) [inline cypress/integration/unit_tests/color_function_util_test.js]: nit: why not just say `let` here and avoid the separate declaration and assignment?

File: docs/README.md [hunk 3]
```diff
@@ -3,10 +3,10 @@
 In `disaster-metadata` => `<year>` => `<disaster>` => `layers` we hold a map of \<asset path : `<data>`>
 
 all `<data>` instances contain the following fields:
-* asset-type `{int}`: corresponds to the LayerType enum
-* display-name `{string}`
-* display-on-load `{boolean}`
-* color-function `{Object}` 
+* assetType `{int}`: corresponds to the LayerType enum
```
Comment (janakdr) [inline docs/README.md]: any interest in converting the docs here into what I did in create_disaster_lib.js: put them with the code?

Comment (juliexxia) [reply docs/README.md]: Yeah, do you mind if I do it in a separate PR?

File: cypress/integration/unit_tests/color_function_util_test.js [hunk 4]
```diff
@@ -65,7 +65,7 @@ describe('Unit tests for color function utility', () => {
     expect(maxMin.is(':visible'));
     expect(maxInput.val()).to.equal('20');
     expect(minInput.val()).to.equal('1');
-    const wings = getCurrentLayers()[0]['color-function']['columns']['wings'];
+    const wings = getCurrentLayers()[0].colorFunction.columns.wings;
```
Comment (janakdr) [inline cypress/integration/unit_tests/color_function_util_test.js]: nit, opt: while wings happens to be a legal suffix here, in general the column can be any string. So could keep this ['wings'] to indicate that. Up to you.

File: cypress/integration/unit_tests/firebase_layers_test.js [hunk 5]
```diff
@@ -2,32 +2,32 @@ import {getLinearGradient} from '../../../docs/firebase_layers.js';
 
 it('creates the correct linear gradients', () => {
   const layer1 = {
-    'color-function': {
-      'current-style': 0,
+    'colorFunction': {
```
Comment (janakdr) [inline cypress/integration/unit_tests/firebase_layers_test.js]: nit: can remove quotes here too

File: cypress/integration/unit_tests/manage_layers_test.js [hunk 6]
```diff
@@ -137,23 +137,21 @@ describe('Unit tests for manage_layers page', () => {
 
     const yellow = 'yellow';
     const singleColor = withColor(
-        createTd(), {color: {'current-style': 2, 'color': yellow}}, property,
-        0);
+        createTd(), {color: {'currentStyle': 2, 'color': yellow}}, property, 0);
```
Comment (janakdr) [inline cypress/integration/unit_tests/manage_layers_test.js]: nit: can get rid of quotes here and a lot below too

**Pull request merged**